### ISSUE TITLE
Issues with persistent classes

### ISSUE THREAD

**Issue by jpdrolet** [opengds/OpenGDS#38]
When trying to make two objects share the same parent object, an error 1446 is generated when the objects are created with persistence. The behavior is as expected for non persistent objects.

![image](https://user-images.githubusercontent.com/37216371/37210495-ea9dea90-2376-11e8-9ee7-0a29a3955f1b.png)

Also, when the class has the persistent pattern with class attributes, the class attributes reference is not created persistent.

**Comment by MikaelHolmstrom** [comment]
You should never connect the "reference in" to the create function.
This is only done inside a Child Create that is inheriting from a Parent.
But I'll test this code, to see what happens.
When you create a Child class it never (in real OO) creates a parent class. It only creates One object of Child. Yes it will use the parent as a template and extend its attribute and method, but you should not see a parent object actually being created.
But in the LV solution we use, we have to create a Parent object to get it working. But you shouldn't try to extract that parent object and share it between multiple children. If you do that you're breaking the OO paradigm.
Can you try to explain what you're trying to achieve?
It looks like you should aggregate the Object 1 instead of inheriting it.

**Comment by jpdrolet** [comment]
I agree that aggregation is a solution. I changed my code to aggregate after seeing the error for persistent classes.
I wanted to share the parent data with many instances of child objects and inheritance allows to use the parent accessors instead of writing wrappers when using aggragation. 

I looked at the code and it is simply that the object on input of the Create method is not passed to the process that creates the persistent references.

Sorry I didn't want to break anything...

Thank you for your time.
